ParseLiveQuery/src/test/java/com/parse/TestParseLiveQueryClient.java

```diff
@@ -390,16 +390,6 @@ public void testEmptySessionTokenOnSubscribe() {
     }
 
     @Test
-    public void testDisconnectOnBackgroundThread() throws Exception {
-        executor.pause();
```

Comment:
Sounds like we should just nuke...can't wait for https://github.com/ParsePlatform/ParseLiveQuery-Android/issues/36 as well